scapy/layers/inet.py

```diff
@@ -387,14 +387,6 @@ def extract_padding(self, s):
         l = self.len - (self.ihl << 2)
         return s[:l],s[l:]
 
-    def send(self, s, slp=0):
```

Comment:
Yes. It dates back to Scapy origin and it is not used. I wonder if IP.route() should also be deleted.